### PR TITLE
Update bentham_app_airflow.py

### PR DESCRIPTION
**Pull request by joeprinold**

### CODE REVIEW

File: bentham_app_airflow.py [hunk 1]
```diff
@@ -26,7 +29,7 @@
     "bentham_app",
     default_args=task_args,
     description="Check s3 for new phone data, then add to database if present.",
-    start_date=datetime(2018, 12, 12, 2),
+    start_date=datetime(2018, 12, 6, 2),
```
Comment (isichei) [inline bentham_app_airflow.py]: I'm not sure it's worth changing the `start_date` can tend to cause issues with airflow.

File: bentham_app_airflow.py [hunk 2]
```diff
@@ -6,13 +6,16 @@
...
 # Define your docker image and the AWS role that will run the image (based on your airflow-repo)
-IMAGE = "593291632749.dkr.ecr.eu-west-1.amazonaws.com/airflow-bentham-app:v0.1.8"
+IMAGE = "593291632749.dkr.ecr.eu-west-1.amazonaws.com/airflow-bentham-app:v0.1.11"
 ROLE = "airflow_bentham_app"
 
 # Task arguments
 task_args = {
     "depends_on_past": False,
     "email_on_failure": True,
+    "retries": 20,
```
Comment (isichei) [inline bentham_app_airflow.py]: When you write reties on a task make sure that your task can run 20 times and get the same result.

For example would run no. 2 be different if the first run failed half way through? 

This makes your life easier in the long run.

File: bentham_app_airflow.py (hunk 1 above)
Comment (joeprinold) [reply bentham_app_airflow.py]: Will change back in future

File: bentham_app_airflow.py (hunk 2 above)
Comment (joeprinold) [reply bentham_app_airflow.py]: Hmm - this isn't really the case as at some point it moves files around etc.

**Pull request merged**